ocaml/xapi/repository.ml

```diff
@@ -1024,3 +1024,128 @@ let get_pool_updates_in_json ~__context ~hosts =
   | e ->
     error "getting updates for pool failed: %s" (ExnHelper.string_of_exn e);
     raise Api_errors.(Server_error (get_updates_failed, []))
+
+let apply ~__context ~host =
+  (* This function runs on slave host *)
+  with_local_repository ~__context (fun () ->
+      let params =
+          [
+            "-y"; "--disablerepo=*";
```

Comment:
Has this code been run through `make format`? I suspect not.

Reply:
when merging to master it will have to be formatted in any case :)

Reply:
Will format all fo these in next PR for refining the code structure.